content/docs/tutorials/handling-errors.mdx

```diff
@@ -0,0 +1,281 @@
+
+  - **transaction submission-related HTTP errors** that occur when performing a `POST` request to Horizon via the `/transaction` endpoint (e.g. via `sdk.submitTransaction` in the JavaScript SDK), whose full enumeration is similarly listed on the [Errors](../../api/errors/result-codes/) page; and 
+
+  - **transient failures** such as the Horizon instance you're relying on going down, your router resetting, AWS hiccupping, etc. which may occur rarely but are nonetheless crucial to handle.
+
+There isn't a universal way to handle HTTP errors: some are the result of user error, some are dynamically recoverable, and some are unavoidable altogether. Transient errors are even trickier to handle due to their rarity and uncertainty, but can be made easier under the assumption that the failing backends will be up to speed again soon. 
+
+Let's dive into each of these separately: we'll go over examples, some common causes, and recommended resolution strategies.
+
+
+## Transaction Submission Errors
+Just like with query errors, there are dozens of possible causes of a transaction submission failure and they all return their own internal result codes. Horizon attempts to wrangle the asynchronous nature of the beast that is Stellar Core into a synchronous flow more familiar to application developers. This requires making assumptions and imposing restrictions, and though it's a simpler model there are still caveats to be aware of. 
+
+Here are some of the common errors encountered when performing a `POST /transactions`:
+
+  - [400 errors, e.g. `tx_bad_seq`](#invalid-sequences)
+  - [Fee / surge pricing errors, e.g. `tx_insufficient_fee`](#surge-pricing)
+  - [504 error, timeout](#timeouts)
+  - [No HTTP response](#outages)
+
+Keep in mind this general rule of thumb: **any response other than `200 OK` or `400 Bad Request` means that the final status of the transaction is unknown.**
+
+
+### 400 Errors
+Generally-speaking, if you get a 400 error during transaction submission, there are two main causes: either your transaction is in an unacceptable format or Stellar Core rejected your transaction. The former case is described by the [`transaction_malformed`](https://developers.stellar.org/api/errors/http-status-codes/horizon-specific/transaction-malformed/) error code and the latter by the [`transaction_failed`](https://developers.stellar.org/api/errors/http-status-codes/horizon-specific/transaction-failed/) code.
+
+Some transaction failures are recoverable; you should look at the `extras.result_code` field in the response message to determine the [root cause](https://developers.stellar.org/api/errors/result-codes/transactions/). One of these recoverable errors is `tx_bad_seq`, where the sequence number on the transaction doesn't match what's expected for the account; another is `tx_insufficient_fee` which is covered [later](#surge-pricing).
```

Comment:
Good point, but I want to avoid enumerating errors if possible, since that should be *more-or-less* researchable from the [`result_codes` page itself](https://developers.stellar.org/api/errors/result-codes/transactions/). I don't think it's possible to be exhaustive. You're right, though; I'll think about this some more!